Analysis/Stan/dagar_seasonal_flexible_narrower_prior.stan

```diff
@@ -59,6 +59,9 @@ data {
   int<lower=0, upper=1> use_weights;
```

Comment:
Merge these into dagar_seasonal_flexible or reset changes i think

Reply:
I've merged narrower prior option but for hte simplex (sum-to-zero constratin), I was unable to conditionally define the eta_tilde (vector or simplex)in the parameter/transformed_parameter part. do you have any thoughts on this?